apiary-extensions-terraform/privileges-grantor/variables.tf

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (C) 2019 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ */
+
+variable "instance_name" {
+  description = "Privilege Grantor instance name to identify resources in multi-instance deployments."
+  type        = "string"
+  default     = ""
+}
+
+variable "subnets" {
+  description = "Subnets in which Lambda will have access to."
+  type        = "list"
+}
+
+variable "security_groups" {
+  description = "Security groups in which Lambda will have access to."
+  type        = "list"
+}
+
+variable "pg_lambda_bucket" {
+  description = "Bucket where the Lambda zip can be found, for example 'bucket_name'. Used together with `pg_jars_s3_key`."
+  type        = "string"
+}
+
+variable "pg_jars_s3_key" {
+  description = "S3 key where zip file is located."
+  type        = "string"
+}
+
+variable "pg_lambda_version" {
+  description = "Version of the Privilege Grantor Lambda."
+  type        = "string"
+}
+
+variable "pg_metastore_uri" {
+  description = "Thrift URI of the metastore to which Lambda will connect to."
+  type        = "string"
+}
+
+variable "metastore_events_sns_topic" {
+  description = "SNS Topic for Hive Metastore events."
+  type        = "string"
+}
+
+variable "metastore_events_filter" {
```

Comment:
The supported format for this variable is same as being done in `hcom-terraform-base`

```
<<EOD 
"CREATE_TABLE","ALTER_TABLE" 
EOD
```

https://stash.hcom/projects/INFRA/repos/hcom-terraform-base/browse/data/lab/account/terraform.tfvars#92